File: JS/AVeryBigSum/Solution1.py
def aVeryBigSum(arr):
    sum = 0
    for int in arr:
        sum += int
    return sum


# TESTS
print(aVeryBigSum([1000000001, 1000000002,
      1000000003, 1000000004, 1000000005]))
# -> 5000000015

File: JS/AVeryBigSum/Solution1.js

const aVeryBigSum = (largeNumbers) => {
  let counter = 0;
  for (let idx = 0; idx < largeNumbers.length; idx++) {
    counter += largeNumbers[idx];
  }
  return counter;
};

// TESTS
console.log(aVeryBigSum([1000000001, 1000000002, 1000000003, 1000000004, 1000000005]));
// -> 5000000015
